src/main/java/james/Ui.java

```diff
@@ -0,0 +1,55 @@
+package james;
+
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the program.
```

Comment:
I like how you included JavaDoc for the class itself as well and not only the methods! 🤩